contracts-periphery/Makefile

```diff
@@ -0,0 +1,39 @@
+# include .env file and export its env vars
+# (-include to ignore error if it does not exist)
+-include .env
+
+all: clean remove install update solc build 
+
+# Install proper solc version.
+solc:; nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions.solc_0_8_10
```

Comment:
(sorry, should have told you there's a lot we don't need in the template I linked you to)

This is something you only need for dapptools, which uses nix, so this can be deleted